src/WinGet.RestSource/Models/Schemas/Information.cs

```diff
@@ -20,8 +21,21 @@ public class Information
         /// </summary>
         public Information()
         {
-            this.SourceIdentifier = ApiConstants.SourceIdentifier;
+            this.SourceIdentifier = string.IsNullOrEmpty(ApiConstants.SourceIdentifier) ? "DefaultSourceIdentifier" : ApiConstants.SourceIdentifier;

```

Comment:
I assume you did this to make it easier to just test a source if it has not had all the proper environment variables set.  

I don't think this is the right way to go.  It bandaids over a configuration issue in a way that will cause distress if/when we have multiple sources all claiming to be the "DefaultSourceIdentifier".  Would we not prefer to detect this configuration error and inform the user of the correct way to configure the reference implementation?

Reply:
Fair point. I have added a check and exception to be thrown for this one.